ads/inabox/inabox-messaging-host.js

```diff
@@ -359,9 +359,12 @@ export class InaboxMessagingHost {
  * @private
  */
 function canInspectWindow_(win) {
+  // TODO: this is not reliable.  The compiler assumes that property reads are
+  // side-effect free.  The recommended fix is to use goog.reflect.sinkValue
+  // but since we're not using the closure library I'm not sure how to do this.
+  // See https://github.com/google/closure-compiler/issues/3156
   try {
-    const unused = !!win.location.href && win['test']; // eslint-disable-line no-unused-vars
-    return true;
+    return !!win.location.href && win['test'];
```

Comment:
I wonder: `win.Object() !== win.Object()`.

Reply:
that could work; @nbeloglazov proposed `(win['test'] || true)` which should also work; I did that in ddad8f6